BOM.txt

```diff
@@ -1,13 +1,18 @@
-PCB BOM
+PCB Build of Materials (BOM)
 
-XH-Series Connectors: 455-2271-ND             3        J1-3
-Power Jack:           CP-002A-ND              1        J4
-1x8 .1" headers:      732-5321-ND             4        J5-8
-2x2 .1" headers:      952-2119-ND             2        J9,10
-Flyback Diodes:       MRA4004T3GOSCT-ND       12       D1-12
-.1uF Filter Caps:     311-1361-1-ND           4        C1-4
-H-Bridge:             497-3624-1-ND           2        IC-1,2
-Indicator LED:        475-1410-1-ND           2        LED 1-2
-Current Limiting R1:  RMCF0805JT270RCT-ND     1        R1
-Current Limiting R2:  P330ADCT-ND             1        R2
-25x25mm heatsink      Custom                  2        N/A
+Part Name                Notes                 PartNumber              Source        QTY      Desiganator
+XH-Series Connectors     Male Connector        455-2271-ND             DigiKey       3        J1-3
+Power Jack               2x5.5MM DC            CP-002A-ND              DigiKey       1        J4
+1x8 .1" headers          2.54mm Pitch          732-5321-ND             DigiKey       4        J5-8
+2x2 .1" headers          2.54mm Pitch          952-2119-ND             DigiKey       2        J9,10
+Flyback Diodes           400V 1A               GS1G-LTPMSCT-ND         DigiKey       12       D1-12
```

Comment:
Changed the diode from one mfg to another as it was regularly kept in stock and was also cheaper. 😈